src/frontends/paddle/src/op/reduce_ops.hpp

```diff
@@ -24,10 +24,26 @@ NamedOutputs reduce_ops(const NodeContext& node) {
         dims = node.get_attribute<std::vector<int32_t>>("dim");
     }
     auto axesNode = default_opset::Constant::create(ngraph::element::i32, {dims.size()}, dims);
-    return node.default_single_output_mapping({std::make_shared<T>(x, axesNode, keep_dim)}, {"Out"});
+    bool scalar_output = !keep_dim;
```

Comment:
Question: Is scalar_output equivalent to !keep_dim?
Is it possible that keep_dim is false, and the output tensor is of rank>1 (the input is of high dims in this situation)?

Reply:
!keep_dim is the initial value of scalar_output. if keep_dim is true, scalar_output is false. If keep_dim is false, it will further check the dims and decide whether it is scalar_output.